markdownextradata/plugin.py

```diff
@@ -16,12 +16,6 @@
 
 CONFIG_KEYS = ["site_name", "site_author", "site_url", "repo_url", "repo_name"]
 
-if sys.version_info[0] >= 3:
-    str_type = str
-else:
```

Comment:
does noone even in legacy long tail land use python 2.* anymore?

Reply:
This is to use the native `str` type for default, removing str_type from mkdocs.utils. Since str_type is used here, our mkdocs sites break with mkdocs >= 1..1

Reply:
hmm but this is testing for python version not mkdocs version what version of python are you using?

Reply:
can you show logs of it breaking please?

Reply:
I'm using Python 3.10.

These are the logs that I obtained from a failing `mkdocs serve`.

```
File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/markdownextradata/plugin.py", line 20, in <module>
    class MarkdownExtraDataPlugin(BasePlugin):
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/markdownextradata/plugin.py", line 26, in MarkdownExtraDataPlugin
    ('data', mkdocs.config.config_options.Type(mkdocs.utils.string_types, default=None)),
AttributeError: module 'mkdocs.utils' has no attribute 'string_types'
```

Reply:
thanks can you please also pop open a python repl in the environment you are using and give me the output of

```
import sys
print(sys.version_info[0])
```

Reply:
I have been using v0.1.1 of this plugin. Just updated it to v0.2.5 and these issues no longer appear.  
Sorry for the inconvenience caused :/

Reply:
no probs! thanks for taking the time to open a PR much appreciate your efforts!